src/Evolution/DgSubcell/Actions/Initialize.hpp

```diff
@@ -122,15 +124,19 @@ struct Initialize {
     const SubcellOptions& subcell_options = db::get<Tags::SubcellOptions>(box);
     const Mesh<Dim>& dg_mesh = db::get<::domain::Tags::Mesh<Dim>>(box);
     const Mesh<Dim> subcell_mesh = fd::mesh(dg_mesh);
-    // Note: we currently cannot do subcell at boundaries, so only set on
-    // interior elements.
+
     const bool cell_is_not_on_external_boundary =
         db::get<::domain::Tags::Element<Dim>>(box)
             .external_boundaries()
             .empty();
 
+    constexpr bool subcell_enabled_at_external_boundary =
+        std::decay_t<decltype(db::get<Parallel::Tags::Metavariables>(
```

Comment:
why can't you just do `Metavariables::SubcellOptions::...`? You have the metavariables as a template parameter to this function, I think that's all you need, then you don't need the tag